mix.exs

```diff
@@ -10,15 +10,15 @@ defmodule Tesla.Mixfile do
       version: @version,
       description: description(),
       package: package(),
-      elixir: "~> 1.10",
+      elixir: "~> 1.16",
       elixirc_paths: elixirc_paths(Mix.env()),
       deps: deps(),
       lockfile: lockfile(System.get_env("LOCKFILE")),
       test_coverage: [tool: ExCoveralls],
       dialyzer: [
         plt_core_path: "_build/#{Mix.env()}",
         plt_add_apps: [:mix, :inets, :idna, :ssl_verify_fun, :ex_unit],
-        plt_add_deps: :project
```

Comment:
the 2 options are here: https://github.com/jeremyjh/dialyxir?tab=readme-ov-file#dependencies